index.html

```diff
@@ -3,22 +3,51 @@
 <head>
     <title>Title</title>
     <link rel="stylesheet" type="text/css" href="main.css">
+    <script src="main.js"></script>
 </head>
 <body>
 <h1>Crypto tracker</h1>
-<ul>
-    <li>
-        <img src="./images/btc.png" class="logo" alt="">
-        Bitcoin
-    </li>
-    <li>
-        <img src="./images/eth.png" class="logo" alt="">
-        Ethereum
-    </li>
-    <li>
-        <img src="./images/xrp.png" class="logo" alt="">
-        Ripple
-    </li>
-</ul>
+<input type="text" id="filterInput" class="filterInput" onkeyup="filterFunction()" placeholder="Search...">
+<div id="myData"></div>
+<script type="text/javascript">
+    fetch('db.json')
+        .then(function (response) {
+            return response.json();
+        })
+        .then(function (data) {
+            appendData(data);
+        })
+        .catch(function (err) {
+            console.log('error: ' + err);
+        });
+
+    function remove() {
+        this.parentNode.parentNode.removeChild(this.parentNode);
```

Comment:
metoda asta e tightly coupled cu structura ta html
gandeste-te la o solutie care sa nu crape daca mai adaugi un element in acel tree